.github/workflows/tgui_recompile.yml

```diff
@@ -0,0 +1,56 @@
+name: Rebuild TGUI
+
+on:
+  repository_dispatch:
+    types: [rebuild-tgui]
+
+# Config
+env:
+  COMMIT_NAME: 'Actions (TGUI Rebuild)'
+  COMMIT_EMAIL: 'action@github.com'
+  DEFAULT_BRANCH: 'master'
+  TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+
+jobs:
+  rebuild:
+    name: Rebuild
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.client_payload.pr_head_full_repo_name }}
+          ref: ${{ github.event.client_payload.pr_head_ref }}
+          fetch-depth: 25
+          token: ${{ env.TOKEN }}
+
+      - name: Setup Author
+        run: |
+          git config --local user.email "${{ env.COMMIT_EMAIL }}"
+          git config --local user.name "${{ env.COMMIT_NAME }}"
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '>=12.13'
+
+      # This only runs if the PR has a merge conflict. Serves to attempt resolving merge conflicts rather than just rebuilding.
+      # Uses a smart little git hack to make a merge commit for just a limited set of files.
+      # Changed tgui/public to browserassets/tgui for goonstation.
+      - name: Conflict Resolution
+        if: ${{ github.event.client_payload.mergeable == false }}
+        run: |
+          git remote add base https://github.com/${{ github.event.client_payload.pr_head_full_repo_name }}.git
+          git fetch base --depth=25 ${{ env.DEFAULT_BRANCH }}
+          git merge -s ours --no-commit base/${{ env.DEFAULT_BRANCH }}
+          git checkout base/${{ env.DEFAULT_BRANCH }} browserassets/tgui
+          git commit -m "TGUI Reset" -a || true
```

Comment:
Why are you using the fork's master branch to resolve conflicts? I know of maybe three people that keep their master in-line with upstream